### PR TITLE
Chapter10: user edit, update, destroy and action

### PR DESCRIPTION
**Pull request by conh-2027**

### CODE REVIEW

**Comment by conh-2027** [comment]
ready

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<h1><%= t ".edit" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= render "form" %>
+    <div class="gravatar_edit">
+      <%= gravatar_for @user %>
+      <%= link_to t(".change"), Settings.gravatar.host, { target: "_blacnk"} %>
+    </div>
+  </div>
+</div>
```
Comment (phucmh-1356) [inline app/views/users/edit.html.erb]: eof

**Comment by conh-2027** [comment]
ready

**Comment by phucmh-1356** [comment]
merged

**Pull request merged**